src/elm/Service/Ticket.elm

```diff
@@ -284,6 +284,9 @@ encodePaymentSelection paymentSelection =
 isScaExemption : PaymentSelection -> Bool
 isScaExemption paymentSelection =
     case paymentSelection of
+        Recurring _ ->
```

Comment:
Yeah, because it only makes sense on recurring, not even NETS when not recurring?

Reply:
I'm pretty sure for non-recurring Nets payments, SCA will be done in the same flow. I suppose we can try to get an exemption though. What do you think?

Reply:
You have probably already taken this into consideration, but when sca was enabled earlier it was enabled for everything except Vipps.

```elm
case paymentSelection of
  NonRecurring PaymentType.Vipps ->
    False

  _ ->
    True
```

Reply:
Yes, we had a short discussion in the office and decided to go with enabling SCA exemption for all Nets payments.